clang/test/Driver/sycl-device-lib-bfloat16.cpp

```diff
@@ -68,6 +68,79 @@
 // RUN:   --sysroot=%S/Inputs/SYCL -### 2>&1 \
 // RUN:   | FileCheck %s -check-prefix=BFLOAT16-FALLBACK-FALLBACK
 
+
+// Test test AOT-DG2 compilation uses native libs + native libs.
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_acm_g10 \
+// RUN:   --sysroot=%S/Inputs/SYCL %s -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16-NATIVE
+
+// Test test AOT-PVC + AOT-DG2 compilation uses native libs + native libs.
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_pvc,intel_gpu_acm_g10 \
+// RUN:   --sysroot=%S/Inputs/SYCL %s -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16-NATIVE-NATIVE
+
+// Test test AOT-PVC + AOT-DG1 compilation uses native libs + native libs.
+// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_dg1,intel_gpu_acm_g10 \
+// RUN:   --sysroot=%S/Inputs/SYCL %s -### 2>&1 \
+// RUN:   | FileCheck %s -check-prefix=BFLOAT16-FALLBACK-FALLBACK
```

Comment:
Comment states use of native/native, but the check is for fallback/fallback.
```suggestion
// Test AOT-PVC + AOT-DG1 compilation uses native libs + native libs.
// RUN: %clangxx -fsycl -fsycl-targets=intel_gpu_dg1,intel_gpu_acm_g10 \
// RUN:   --sysroot=%S/Inputs/SYCL %s -### 2>&1 \
// RUN:   | FileCheck %s -check-prefix=BFLOAT16-FALLBACK-FALLBACK
```